editor/editor_properties.h

```diff
@@ -62,6 +62,7 @@ class EditorPropertyText : public EditorProperty {
 	static void _bind_methods();
 
 public:
+	virtual void _set_read_only(bool p_read_only) override;
```

Comment:
This should be in `protected` field (next to `_bind_methods()`). Same in all classes.